content/doc/book/pipeline/syntax.adoc

```diff
@@ -1290,7 +1289,7 @@ making it an ideal choice for simpler continuous delivery pipelines. Scripted
 provides very few limits, insofar that the only limits on structure and syntax
 tend to be defined by Groovy itself, rather than any Pipeline-specific systems,
 making it an ideal choice for power-users and those with more complex
-requirements. As the name implies, Declarative Pipeline is encourages a
+requirements. As the name implies, Declarative Pipeline encourages a
```

Comment:
"Scripted and Declarative use different syntax and provide different levels of flexibility.
Declarative has a strict and pre-defined structure;
this limits what is available to the user
but also makes it an ideal choice for simpler continuous delivery pipelines.
Scripted is limited only by Groovy syntax limits,
not by any Pipeline-specific systems.
This makes it an ideal choice for power users
and those with more complex requirements.